vegacapsule/genesis.tmpl

```diff
@@ -196,7 +196,7 @@
 		"market.fee.factors.makerFee": "0.004",
 		"market.liquidity.stakeToCcyVolume": "0.3",
 		"market.liquidity.targetstake.triggering.ratio": "0.7",
-    "market.liquidity.providersFeeCalculationTimeStep": "5s",
+		"market.liquidity.providersFeeCalculationTimeStep": "5s",
```

Comment:
this one was breaking something at one point when it got removed

Reply:
that was an SLA  parameter that has been removed between tags.
Reason being it was a duplicate, we had it as a network parameter and  and market parameter. It was removed from the market fields.

Reply:
let's give it a go for main/develop consistency. 
if it breaks again on develop capsule we know we need to remove it.